src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/SummaryJavadocCheck/context/Test.java

```diff
@@ -0,0 +1,10 @@
+package TreeWalker.javadoc.SummaryJavadoc;  // violation without filter on line 4
+
+public class Test {
+    /**
```

Comment:
make it:
```
/**@tag // violation without filter 
*/
```

please apply the same for others

Reply:
@romani if we change it:

```
/**@tag // violation without filter 
*/
```
then, it will change line 4, and we can not reflect `context` strategy.

Reply:
@HuGanghui please play with https://checkstyle.sourceforge.io/config_javadoc.html#SummaryJavadoc options and make code look like this:
```
/** // violation without filter
*
*/
```
It should be possible to have `// violation without filter` on a proper line and have a violation from the check.

Reply:
@rdiachenko  I try it and not find a good option to solve this problem.

Reply:
@HuGanghui this is not the first time you didn't provide any details of your attempts, why?

This works fine on my local, please apply to all strategies:
```
$ cat defaultContext.patch 
diff --git a/Test.java b/Test.java
index 5319710..768465a 100644
--- a/Test.java
+++ b/Test.java
@@ -2,7 +2,6 @@ package TreeWalker.javadoc.SummaryJavadoc;

 public class Test {
     /** // violation without filter
-     * {@inheritDoc}
      *
      */
     public String ValidFunction(){
```
```
$ cat Test.java 
package TreeWalker.javadoc.SummaryJavadoc;

public class Test {
    /** // violation without filter
     *
     */
    public String ValidFunction(){
        return "";
    }
}
```
```
$ cat expected.txt 
Test.java:4: First sentence of Javadoc is missing an ending period.
```

Reply:
@rdiachenko sorry, It's my fault, I forgot to provide details. And I try this violation `First sentence of Javadoc is missing an ending period.`, but I don't think it can reflect `context` strategy, because `missing an ending period` is just caused by line 4 itself, not by deleting `{@inheritDoc}`. 

```
     /**
-     * {@inheritDoc}
+     *
      */
```
this change will cause violation `Test.java:4: Summary javadoc is missing.` and can reflect `context` strategy, but them we can not find a good way to add `// violation without filter` in it, because if we add this sentence, them it will be summary Javadoc and no violation.

Reply:
@HuGanghui 
```
$ cat defaultContext.patch 
diff --git a/Test.java b/Test.java
index 5319710..768465a 100644
--- a/Test.java
+++ b/Test.java
@@ -1,10 +1,7 @@
 package TreeWalker.javadoc.SummaryJavadoc;

 public class Test {
-    /**
-     * {@inheritDoc}
-     *
-     */ // violation without filter
+    /** */ // violation without filter
     public String ValidFunction(){
         return "";
     }
```
```
$ cat Test.java 
package TreeWalker.javadoc.SummaryJavadoc;

public class Test {
    /** */ // violation without filter
    public String ValidFunction(){
        return "";
    }
}
```
```
$ cat expected.txt 
Test.java:4: Summary javadoc is missing.
```

Reply:
@rdiachenko this example can be solved by `patchedline` strategy, not need `context` strategy, so it's ok that we use this example ?

Reply:
Yes, all `violation without filter` comments must be on the line with violation

Reply:
ok, done